packages/form/src/form-item.vue

```diff
@@ -45,13 +45,14 @@
 import type { PropType } from 'vue'
 import { computed, defineComponent, getCurrentInstance, inject, nextTick, onBeforeUnmount, onMounted, provide, reactive, ref, toRefs, watch } from 'vue'
 import { NOOP } from '@vue/shared'
-import AsyncValidator, { RuleItem } from 'async-validator'
+import AsyncValidator from 'async-validator'
 import LabelWrap from './label-wrap'
 import { getPropByPath, useGlobalConfig } from '@element-plus/utils/util'
 import { isValidComponentSize } from '@element-plus/utils/validators'
 import mitt from 'mitt'
 import type { ElFormContext, ValidateFieldCallback } from './token'
 import { elFormEvents, elFormItemKey, elFormKey } from './token'
+import { FormItemRule } from './form.type'
```

Comment:
Would you mind adding `type` keyword right after `import`?
Also please move the code at line#45 to the bottom for code styling, also line#53. Thanks!